### PR TITLE
Added Support --perf-output in CLI

### LINKED ISSUE #102: Support `--perf-output` in CLI
related #85 

You can generate a `perf.json` to inspect the performance statistic of wakaru by using `--perf` flag. But where would this file be generated is a bit awkward. By default, it will be `{outDir}/perf.json`. But users are allowed to override the output folder of features by using `--unpacker-output` and `--unminify-output`. In this case, a common base directory will be the folder where we generate the perf file.

Having a `--perf-output` or even `--perf-path` can be useful for power-users who manage and process a lot of samples.

### PR DESCRIPTION
**Pull request by AhmadFaraz-crypto**
Closes #102

### CODE REVIEW

File: packages/cli/src/cli.ts [hunk 1]
```diff
@@ -35,6 +35,7 @@ import type { Module } from '@wakaru/unpacker'
 enum Feature {
     Unpacker = 'Unpacker',
     Unminify = 'Unminify',
+    Perf = 'Perf',
```
Comment (pionxzh) [inline packages/cli/src/cli.ts:38]: I think this one is not needed?

File: packages/cli/src/cli.ts [hunk 2]
```diff
@@ -480,14 +487,15 @@ async function nonInteractive(features: Feature[], {
...
+    const perfOutputBase = _perfOutput || (singleFeature
         ? features.includes(Feature.Unpacker) ? unpackerOutput : unminifyOutput
-        : findCommonBaseDir([unpackerOutput, unminifyOutput]) ?? outputBase
+        : findCommonBaseDir([unpackerOutput, unminifyOutput]) ?? outputBase)
     const perfOutputPath = path.join(perfOutputBase, 'perf.json')
 
     const outputPathsToCheck = []
     if (features.includes(Feature.Unpacker)) outputPathsToCheck.push(unpackerOutput)
     if (features.includes(Feature.Unminify)) outputPathsToCheck.push(unminifyOutput)
+    if (features.includes(Feature.Perf)) outputPathsToCheck.push(perfOutputPath)
```
Comment (pionxzh) [inline packages/cli/src/cli.ts:498]: So this one should be 
```suggestion
    outputPathsToCheck.push(perfOutputPath)
```

File: packages/cli/src/cli.ts (hunk 1 above)
Comment (AhmadFaraz-crypto) [reply packages/cli/src/cli.ts]: Got it. I removed it and just pushed the updated code.

**Comment by pionxzh** [comment]
LGTM! Thanks

**Review by pionxzh — APPROVED** [review]

**Comment by AhmadFaraz-crypto** [comment]
> LGTM! Thanks

Thanks 🙂! Let me know if there is anything else to contribute. I will be happy to help you.

**Comment by pionxzh** [comment]
I think #82 and #116 would be easier to work on. But this two requests still have some uncertain part that might need further discussion. You can put some of your opinion and maybe implement part of them.

**Comment by AhmadFaraz-crypto** [comment]
Sure, let me have a look.

**Comment by pionxzh** [comment]
I have released CLI `v0.0.10` with these changes.

**Comment by AhmadFaraz-crypto** [comment]
Thanks for letting me know. I will check this version.

**Comment by AhmadFaraz-crypto** [comment]
I will look into another issue.

**Pull request merged**